src/containers/org/MemberRolesContainer.js

```diff
@@ -11,6 +12,9 @@ import { isRoleAssignedToMember } from './utils';
 
 import { StackGrid } from '../../components';
 import { selectCurrentUserOrgOwner } from '../../core/redux/selectors';
+import { INITIAL_PAGINATION_STATE, paginationReducer } from '../../utils/stateReducers/pagination';
+
+const MAX_PER_PAGE = 10;
```

Comment:
No change required, just a note. I know I've been doing this `MAX_PER_PAGE` thing in a lot of places, but I probably shouldn't have because there exists a `MAX_HITS_10` constant in `src/core/search/constants/index.js`.